bson/src/main/org/bson/RawBsonDocument.java

```diff
@@ -349,7 +349,8 @@ private BsonBinaryReader createReader() {
         return new BsonBinaryReader(new ByteBufferBsonInput(getByteBuffer()));
     }
 
-    private BsonDocument toBsonDocument() {
```

Comment:
Had to change the name of this private method to avoid name conflict with the new method.